lib/application.js

```diff
@@ -520,6 +520,19 @@ exports.loadConfig = (persistent, opts) => {
       .then(deepFreeze);
 };
 
+// workerk don't need default server, deprecation warnings, or package manager support
+exports.loadConfigWorker = (persistent, opts) => {
```

Comment:
It's like 300ms faster on our app than `loadConfig` due to dropped unneeded steps